iceoryx_utils/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ if(GTest_FOUND) # only GTest_FOUND, just in case someone want's to use iceoryx_u
         testutils/mocks/mqueue_mock.cpp
         testutils/mocks/time_mock.cpp
         testutils/timing_test.cpp
+        testutils/compile_test.cpp
```

Comment:
since we had a completely different idea how to solve this, would it be a good idea to remove this?